dev/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM php:8.1-fpm
+
+RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y git libicu-dev nodejs \
+    nginx libpng-dev libzip-dev libwebp-dev unzip zip libxml2-dev libjpeg62-turbo-dev zlib1g-dev \
+    libfreetype6 libfreetype6-dev
+
+RUN docker-php-ext-install opcache \
```

Comment:
Do we really need the opcache stuff? It seems like added complication and build time in this Dockerfile.

I understand it can be useful for performance, but it seems pretty performant without this to me, and this is just for a development environment anyway.

Reply:
I tested it now and it's really not necessary, it was removed